suites/pacific/cephadm/tier_2_cluster_scaleup_scaledown.yaml

```diff
@@ -0,0 +1,624 @@
+tests:
```

Comment:
please add the test case description and details in the comment part.

Reply:
Done.